test/integration/concrete/lockup/withdraw-max-and-transfer/withdrawMaxAndTransfer.t.sol

```diff
@@ -112,11 +112,11 @@ abstract contract WithdrawMaxAndTransfer_Integration_Concrete_Test is
         emit Transfer({ from: users.recipient, to: users.alice, tokenId: defaultStreamId });
 
         // Make the max withdrawal and transfer the NFT.
-        lockup.withdrawMaxAndTransfer({ streamId: defaultStreamId, newRecipient: users.alice });
+        uint128 expectedWithdrawnAmount =
+            lockup.withdrawMaxAndTransfer({ streamId: defaultStreamId, newRecipient: users.alice });
 
         // Assert that the withdrawn amount has been updated.
         uint128 actualWithdrawnAmount = lockup.getWithdrawnAmount(defaultStreamId);
-        uint128 expectedWithdrawnAmount = withdrawAmount;
```

Comment:
Since this assertion is to verify that the "`getWithdrawnAmount` amount has been correctly updated", it should be be compared against `lockup.withdrawableAmountOf(defaultStreamId)`.

And then we should have a separate assertion to verify that the "returned" value is indeed same as `lockup.withdrawableAmountOf(defaultStreamId)` since thats what it is expected to return.